sov-modules/sov-modules-impl/accounts/src/lib.rs

```diff
@@ -1,3 +1,5 @@
+#![feature(adt_const_params)]
```

Comment:
This is an unstable feature. I used it to pass HRP to the serialization/deserialization code `const HRP: &'static str`. It works here fine.

If not like this we can:
1. Hardcode the HRP making the serialization/deserialization only work for addresses (we can invoke general methods from it)
2. In response have a custom struct instead of just `Vec<u8>` that would contain the HRP as well as the data
3. Create a custom macro (overkill here)